build.gradle.kts

```diff
@@ -51,7 +51,7 @@ allprojects {
 }
 
 group = "exchange.dydx.abacus"
-version = "1.7.74"
+version = "1.8.0"
```

Comment:
The convention is to update the minor version number when there is a breaking change (meaning the client code will need to be updated after bumping to this Abacus version).

In your case, it looks like the change is non-breaking.  Recommend just updating the patch number (or leave it to the Github Action to update the version).

Reply:
Oh nevermind.. a field name got change..  Yeah.. then it makes sense to update the minor version... Up to you :-)